README.md

```diff
@@ -5,14 +5,19 @@
 
 This repository contains the materials for D-Lab [WorkshopName] workshop. Prior experience with [PriorWorkshopWithLink] is assumed.
 
-## Workshop Goals
+## Learning Objectives
 
 In this workshop, we provide [WorkshopGoal]. First, we’ll cover [Aim1]. Then, we’ll explore [Aim2]. 
```

Comment:
What is the difference between Aims and Learning Objectives?

Reply:
I see Workshop Goals as a quick summary paragraph with goals and what you need. The Learning Objectives give more in-depth info on what to expect and not expect.